test/ctl/test.cpp

```diff
@@ -0,0 +1,20 @@
+#include <common/base.hpp>
+#include <ctl/ctl.h>
+#include <ctl/ctl_debug.h>
+
+using namespace umf_test;
+
+TEST_F(test, ctl_debug) {
+    initialize_debug_ctl();
+    auto ctl_handler = get_debug_ctl();
+    ctl_load_config_from_string(ctl_handler, NULL,
+                                "debug.heap.alloc_pattern=1");
+
+    int value = 0;
+    ctl_query(ctl_handler, NULL, CTL_QUERY_PROGRAMMATIC,
+              "debug.heap.alloc_pattern", CTL_QUERY_READ, &value);
+    ASSERT_EQ(value, 1);
```

Comment:
we definitely need more complex tests

Reply:
I will port CTL tests from PMDK repository, I've added extra test to cover most of CTL functionality.
Before:
![image](https://github.com/user-attachments/assets/90935888-3545-4fcb-a720-87381b90f5ce)
After:
![image](https://github.com/user-attachments/assets/5d8a6bd0-7dac-4ba0-9f6d-482595957fe3)

Reply:
OK, as for the first version